src/components/SupportRequest.js

```diff
@@ -208,6 +211,19 @@ const SupportRequest = _.flow(
             onChange: v => this.setState({ email: v })
           })
         ])]),
+        h(IdContainer, [id => h(Fragment, [
```

Comment:
For an `IdContainer` to work, the `id` needs to be attached to a label and also a target element. Radio buttons have their own built in labels though, so I don't think we need another one in this case.